CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -195,6 +199,22 @@ static function X2MissionSourceTemplate GetCovertMissionSource(X2CovertMissionIn
 	return X2MissionSourceTemplate(StratMgr.FindStrategyElementTemplate(MissionInfo.MissionSource));
 }
 
+static function StrategyCost GetExfiltrationCost(XComGameState_CovertAction CovertAction)
+{
+	local StrategyCost ExfiltrateCost;
+	local ArtifactCost IntelCost;
+	local TDateTime CurrentTime;
+
+	CurrentTime = class'XComGameState_GeoscapeEntity'.static.GetCurrentTime();
+
+	IntelCost.Quantity = default.BASE_INTEL_COST + class'X2StrategyGameRulesetDataStructures'.static.DifferenceInDays(CurrentTime, CovertAction.StartDateTime) * default.INTEL_COST_MULTIPLIER;
```

Comment:
Get difference in hours (and then divide by 24) so that the player isn't penalized for clicking one second too late